lib/resources/token.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var token = exports; exports.constructor = function token() {};
+
+var crypto = require('crypto');
+
+var jwt = require('jsonwebtoken');
+
+token.initialize = function(secret) {
+  this._secret = secret;
+};
+
+token.generate = function(cb) {
+  var self = this;
+
+  crypto.randomBytes(32, function(err, buffer) {
+    if (err) {
+      return cb(err);
+    }
+
+    var token = jwt.sign({ id: buffer.toString('hex') }, self._secret);
+    cb(null, token);
```

Comment:
should probably have this return a rpomise and wrap the `crypto.randomBytes` since promises are used thorughout the model code